rainwave/user.py

```diff
@@ -280,6 +280,9 @@ def remove_request(self, song_id):
 	def clear_all_requests(self):
 		return db.c.update("DELETE FROM r4_request_store WHERE user_id = %s", (self.id,))
 
+	def clear_all_requests_on_cooldown(self):
+		return db.c.update("DELETE FROM r4_request_store JOIN r4_song_sid ON r4_song_sid.song_id = r4_request_store.song_id AND r4_song_sid.sid = r4_request_store.sid WHERE user_id = %s AND song_cool_end - extract(EPOCH FROM now()) > 20 * 60", (self.id,))
```

Comment:
Instead of `EPOCH FROM now()` can you use Python timestamping please?

something more like this:
```
from time import time as timestamp

return db.c.update("DELETE ... song_cool_end - %s", (self.id, timestamp() - (20 * 60))`

Reason our timestamps are in Python and renamed to `timestamp` rather than `time.time` is there was a plan with JF once to run experiments where we could change time at will to test different cooldown parameters. :)  It'd be nice to keep the the code consistent there. :D